functions/index.js

```diff
@@ -38,6 +38,94 @@ exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
   });
 });
 
+exports.initSettings = functions.https.onRequest(async (req, res) => {
+  return cors(req, res, async () => {
+    console.log("Initializing settings");
+    try {
+      const querySnapshot = await admin.firestore().collection("app_settings").doc("default").create({
```

Comment:
Could we just init this if the document doesn't exist e.g. in updateSettings? That way we can avoid having to call the initSettings function separately!